CSS/index.css

```diff
@@ -334,12 +325,180 @@ p2 {
   margin-top: 2%;
 }
 
+#footerRight {
+  width: 70%;
+  margin-left: 0%;
+  display: flex;
+  justify-content: flex-end;
+}
+
 #socialLogos {
-  margin-right: 3%;
-  margin-top: 1%;
   cursor: pointer;
+  width: 20%;
+  /* position: absolute; */
+}
+
+#socialImage {
+  width: 33%;
+  height: 60%;
+  margin-left: 2%;
+  margin-top: 2%;
 }
 
 p5 {
   color: #4282d6;
 }
+
+/* MOBILE DESIGN */
+
+@media only screen and (max-width: 800px) {
```

Comment:
Great job with your queries. Excellent show of improvement overall on responsive code